src/components/Explore/Explore.tsx

```diff
@@ -134,9 +136,10 @@ const Explore: React.FunctionComponent<{
       setCurrentMetric(sharedMetric);
     }
 
-    const onSelectCurrentMetric = (newMetric: number) => {
-      const newMetricName = metricLabels[newMetric];
-      setCurrentMetric(newMetric);
+    const onSelectCurrentMetric = (newMetricIndex: number) => {
+      const newMetricName = metricLabels[newMetricIndex];
+      const metric = EXPLORE_METRICS[newMetricIndex];
```

Comment:
Rather than assuming the menu index will match the metric enum value, we look up the metric enum value.

Reply:
ahhh yes